Emitron/emitronScreenshots/EmitronScreenshots.swift

```diff
@@ -42,7 +42,7 @@ class EmitronScreenshots: XCTestCase {
     
     // Reset filters
     app.buttons.matching(identifier: "Filter Library").element.tap()
-    app.buttons.matching(identifier: "Clear All").element.tap()
+    app.buttons.matching(identifier: "Reset filters").element.tap()
```

Comment:
Nit - filters should be capitalised

```suggestion
    app.buttons.matching(identifier: "Reset Filters").element.tap()
```